worker/src/worker.rs

```diff
@@ -332,21 +330,8 @@ impl Worker {
         &self,
         tx_reconfigure: &watch::Sender<ReconfigureNotification>,
         tx_primary: Sender<WorkerPrimaryMessage>,
-        rx_worker_request: types::metered_channel::Receiver<(Vec<BatchDigest>, PublicKey)>,
         rx_worker_processor: types::metered_channel::Receiver<Batch>,
-        network: anemo::Network,
     ) -> Vec<JoinHandle<()>> {
```

Comment:
Maybe we can just return `JoinHandle<()>` now.

Reply:
I actually made that change originally and then put it back, just in case we add another task again here later. If we think that will never happen, then this doesn't even need to be a separate function. I can inline the one single call to `Processor::spawn` in `Worker::spawn` just as easily - wdyt?

Reply:
I don't have strong opinion either way. Since this has less delta I think we can definitely keep the current way.